develop/reference/articles/05-theme-gulp-tasks.markdown

```diff
@@ -1,32 +1,39 @@
-# Theme Gulp Tasks [](id=theme-gulp-tasks)
+# Liferay Theme Generator Reference Guide
 
-Theme projects created using the [Liferay Theme Generator](/develop/tutorials/-/knowledge_base/7-0/themes-generator)
-have a `gulpfile.js` file that provides several [gulp](https://www.npmjs.com/package/gulp)
-tasks you can execute to manage and deploy your theme.
+This reference guide provides additional configuration information for the 
+[Liferay Theme Generator](/develop/tutorials/-/knowledge_base/7-0/themes-generator), 
+such as the available gulp tasks for generated themes, optional flags for the 
+generator, etc.
 
-Here are the gulp tasks you can execute:
+## Theme Gulp Tasks [](id=theme-gulp-tasks)
 
-*  `build`: generates the base theme files, compiles Sass into CSS, and
-    zips all theme files into a WAR file that you can deploy to a Liferay
-    server. 
+Theme projects created using the 
```

Comment:
@Robert-Frampton I know the Server Manager Web plugin and --live deploy flag didn't work for 7.0. Will this be fixed for 7.1?

Reply:
No, I don't that plugin will ever be used in future releases.